WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalSuccessWithoutEmail.swift

```diff
@@ -66,5 +64,10 @@ private extension CardPresentModalSuccessWithoutEmail {
             "Print receipt",
             comment: "Button to print receipts. Presented to users after a payment has been successfully collected"
         )
+
+        static let saveReceiptAndContinue = NSLocalizedString(
+            "Save receipt and continue",
+            comment: "Button when the user does not want to print or email receipt. Presented to users after a payment has been successfully collected"
```

Comment:
nit: I think there's just print action here 🙂 
```suggestion
            comment: "Button when the user does not want to print receipt. Presented to users after a payment has been successfully collected"
```